lib/components/team/Cluster.js

```diff
@@ -61,9 +67,10 @@ class Cluster extends AutoRefreshComponent {
       const state = copy(this.state)
       const cluster = state.cluster
       await apiRequest(null, 'delete', `/teams/${team}/clusters/${cluster.metadata.name}`)
-      await apiRequest(null, 'delete', `/teams/${team}/gkes/${cluster.metadata.name}`)
-      state.cluster.deleted = true
+      cluster.status.status = 'Deleting'
+      cluster.metadata.deletionTimestamp = new Date()
```

Comment:
No longer deleteing the resource from /teams/${team}/gkes/... ? Is that correct?

Reply:
Yep that's taken care of Kore 👍